news-aggregation/services/NewsDeliveryService.ts

```diff
@@ -0,0 +1,45 @@
+import {
```

Comment:
This file is not needed I guess, since we will be handling the logic in the scheduled jobs